pytest_testdox/plugin.py

```diff
@@ -36,24 +36,26 @@ def should_enable_plugin(config):
 
 @pytest.mark.trylast
 def pytest_configure(config):
+    # Register testdox markers even though the plugin is not involved
```

Comment:
I think this comment is not necessary. People need to have the context of what is described in issue #31 to get its meaning.

Reply:
Yes, you right. Remove it.